### ISSUE TITLE
[Designer][Bug 40018958](https://microsoft.visualstudio.com/OS/_workitems/edit/40018958): [ExpenseReport.json]: Screen reader is not announcing table and column header information for the Data (Layout) table present under 'ExpenseReport.json' card.

### ISSUE THREAD

**Issue by aaagrawal** [microsoft/AdaptiveCards#7613]
### Problem Description

[Bug 40018958](https://microsoft.visualstudio.com/OS/_workitems/edit/40018958): [ExpenseReport.json]: Screen reader is not announcing table and column header information for the Data (Layout) table present under 'ExpenseReport.json' card.

Test Environment:
Test Environment:
OS: Win11 (25136.1000)
UWP App version: 3.0.2206.2001
Tool: Accessibility Insights for Win32
 
Pre-Requisite: Launch the Adaptive Card application.

Repro steps:
1. Launch the Adaptive Card application and Enable Narrator.
2. Adaptive Card UWP Visualizer screen starts appearing. 
3. Navigate to ExpenseReport.Json button by using Tab key and activate it.
4. Expense approval adaptive card will be appeared on right sight of the pane.
5. Navigate to Expense approval card by using Tab key.
6. Switch narrator into scan mode.
7. Navigate to data.

Actual Result:
Screen reader is not announcing table and column header information when navigate inside a layout table under 'ExpenseReport.json' card.

Expected result:
Screen reader should announce table and column header information followed by the announcement of a table/grid or help user understand the proper UI structure.
User Experience:
Screen reader users will be impacted if UI structure is not conveyed properly via AT's

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Card JSON

```JSON
Designer Bug
```

**Comment by aaagrawal** [comment]
_Linked to Azure Boards Test Plans_
[### **Design**-Test Case: 28646792 Navigate to "ExpenseReportwithLabels.JSON" button](https://microsoft.visualstudio.com/DefaultCollection/OS/_workitems/edit/28646792)

**Comment by Rishikesh6521** [comment]
**UI Changed,** 

Currently the table is not present or provided on the application. It is present in the form of button controls. Please refer the screenshot below:

![Image](https://github.com/user-attachments/assets/3a2db9fa-db44-4eca-8ff2-6694241ab504)
